pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/steps/CredentialWrapperStepTest.java

```diff
@@ -120,7 +120,7 @@ public static void setup() throws Exception {
         folderStore.addCredentials(Domain.global(), c);
 
         File crt = new File(CredentialWrapperStepTest.class.getResource("dummy.p12").toURI());
-        CertificateCredentialsImpl certCred = new CertificateCredentialsImpl(CredentialsScope.GLOBAL, "certCred1", "sample", "test", 
+        CertificateCredentialsImpl certCred = new CertificateCredentialsImpl(CredentialsScope.GLOBAL, "certCred1", "sample", "password", 
```

Comment:
double checked this was the correct password with `openssl pkcs12 -legacy -info -in ./src/test/resources/org/jenkinsci/plugins/pipeline/modeldefinition/steps/dummy.p12`